core/src/test/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -143,7 +143,6 @@ private void slowDownRecovery(ByteSizeValue shardSize) {
                 .setTransientSettings(Settings.builder()
                                 // one chunk per sec..
                                 .put(RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC, chunkSize, ByteSizeUnit.BYTES)
-                                .put(RecoverySettings.INDICES_RECOVERY_FILE_CHUNK_SIZE, chunkSize, ByteSizeUnit.BYTES)
```

Comment:
I was afraid we will over sleep trying to transfer 512KB at one go. Double checking I see that we only transfer single files so we will never get a 512KB chunk in the test. All good then.